support-frontend/assets/helpers/user/user.js

```diff
@@ -2,13 +2,13 @@
 
 // ----- Imports ----- //
 
-import { routes } from 'helpers/routes';
-import * as cookie from 'helpers/cookie';
-import { get as getCookie } from 'helpers/cookie';
-import { getSession } from 'helpers/storage';
+import { routes } from 'helpers/urls/routes';
+import * as cookie from 'helpers/storage/cookie';
+import { get as getCookie } from 'helpers/storage/cookie';
```

Comment:
```suggestion
import * as cookie, { get as getCookie } from 'helpers/storage/cookie';
```

Reply:
The `import * as` syntax does not seem to allow this, unfortunately.